lib/manager/maven/extract.spec.ts

```diff
@@ -123,6 +124,112 @@ describe('manager/maven/extract', () => {
         packageFile: null,
       });
     });
+    it('extract dependencies from any XML position with Central Maven', () => {
+      const res = extractPackage(simpleCentralContent);
+      expect(res).toMatchSnapshot({
+        deps: [
+          {
+            depName: 'org.example:parent',
```

Comment:
I think you should add the empty registry urls field, otherwise it can have any value.